src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -1458,6 +1483,89 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		const currentAgent = this.parsedInput.parts.find(part => part instanceof ChatRequestAgentPart);
 		this.agentInInput.set(!!currentAgent);
 	}
+
+	/**
+	 * Gets a list of all tools specified in the provided prompt files.
+	 */
+	public async getPromptFileToolsMetadata(
```

Comment:
Could be private?

Reply:
Good catch, will update in the next PR 👍